modules/core/shared/src/main/scala/lucuma/core/model/Proposal.scala

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.model
+
+import cats.Eq
+import cats.Order._
+import cats.syntax.all._
+import eu.timepit.refined.auto._
+import eu.timepit.refined.cats._
+import eu.timepit.refined.types.string.NonEmptyString
+import lucuma.core.enum.TacCategory
+import lucuma.core.enum.ToOActivation
+import lucuma.core.math.units.IntPercent
+import monocle.Focus
+
+import scala.collection.immutable.SortedMap
+
+final case class Proposal(
+  title:         Option[NonEmptyString],
+  proposalClass: ProposalClass,
+  category:      Option[TacCategory],
+  toOActivation: ToOActivation,
+  abstrakt:      Option[NonEmptyString],
+  partnerSplits: SortedMap[Partner, IntPercent]
```

Comment:
FWIW, my goal was to allow the PI to specify the percentages so that they don't have to worry about adjusting hours per partner when the total time request invariably changes because they added targets or changed their constraints.

Reply:
This makes sense, particularly considering that the time request itself will just be calculated from the included observations instead of typed in by the user.

Reply:
As the program changes during phase 2 and execution, we'll lose the initial time request (along with which observations were initially proposed and how long they were initially considered to last).  Is that an issue?  If so, I suppose we might need to store a snapshot of the information at proposal time.